python/data/chap4-7/eflr/frames-and-channels/README.rst

```diff
@@ -0,0 +1,24 @@
+Frames and Channels
+===================
+
+Specific tests for channels and frames
+
+================================================ ==================================================
+Filename                                         Description
+================================================ ==================================================
+
+duplicated.dlis                                  Frame contains two same channels
```

Comment:
"A Frame containing two Channels with the same mnemonic, origin and copynumber" maybe?